src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
To avoid cache issue on the start project, you can remove all of part related to serviceWorker